ph5/utilities/validation.py

```diff
@@ -128,70 +134,48 @@ def check_response_info(info, ph5, unique_filenames_n_i, checked_data_files,
             das or sensor if loaded or None if none is loaded
     """
     Response_t = ph5.get_response_t_by_n_i(info['n_i'])
+    header = ("array {0}, station {1}, channel {2}, "
+              "response_table_n_i {3}: ").format(info['array'],
+                                                 info['sta'],
+                                                 info['cha_id'],
+                                                 info['n_i'])
     if Response_t is None:
-        errmsg = "No response entry for n_i=%s." % info['n_i']
-        addError(errmsg, errors, logger)
-        return False
+        errmsg = ("%sresponse_t has no entry for n_i=%s"
+                  % (header, info['n_i']))
+        return False, errmsg
     if info['n_i'] == -1:
         # metadata no response signal
-        return False
+        errmsg = ("%sMetadata response with n_i=-1 has no response data."
+                  % header)
+        return False, errmsg
 
-    m_file = None
+    d_file = None
+    s_file = ''
     # check resp file from metadata
```

Comment:
This logic seems to be different. Is there a way to test the new logical path and verify that it is working as expected?